### PR TITLE
[Feature 9] openMonthMenu and openMainMenu

### LINKED ISSUE #22: [Feature 9] Month Menu Button Reactions
As buttons are clicked on Month Menu, different features including add/edit expenses, add/edit categories and generate summary will activate.

### PR DESCRIPTION
**Pull request by YinLingithub**
Made implementation changes in NewMonthV that updates MainMenuV when a new MonthlyStorage is create, and changes in MonthMenuV to open GUI when
1. New MonthlyStorage is created
2. user selects to view a saved Month
3. user add/edit Expense/Category

There are not tests for this pull request as changes are made in view classes. 
Javadoc comments should be updated.

### CODE REVIEW

File: src/main/java/views/monthly_menu/MonthMenuV.java [hunk 1]
```diff
@@ -16,15 +22,76 @@
...
+    public void actionPerformed(ActionEvent event) {
+        if (event.getSource()==addExpense){
+            // call add expense
+        } else if (event.getSource()==editExpense) {
+            // call edit expense
+        } else if (event.getSource()==addCategory) {
+            // call add category
+        } else if (event.getSource()==editCategory) {
+            // call edit category
+        } else if (event.getSource()==generateSummary) {
```
Comment (katarinaavucic) [inline src/main/java/views/monthly_menu/MonthMenuV.java:69]: Looks great!

File: src/main/java/views/monthly_menu/MonthMenuV.java [hunk 2]
```diff
@@ -36,15 +103,8 @@ public class MonthMenuV implements ActionListener {
...
-     * @param controller the controller class to get output data
-     * @param session the SessionStorage holding the required MonthlyStorage
-     * @param monthID the monthID of the required MonthlyStorage
      */
-    public MonthMenuV(views.monthly_menu.UpdateViewC controller, SessionStorage session, int monthID) {
-        this.controller = controller;
-        this.session = session;
-        this.monthID = monthID;
-
+    private void createMonthMenuView(){
```
Comment (katarinaavucic) [inline src/main/java/views/monthly_menu/MonthMenuV.java:107]: Good use of a private method here!

**Review by katarinaavucic — APPROVED** [review]
Looks good so far! Approving so Ari can get started on combining his portion of code.

**Review by aricasas — APPROVED** [review]
Approved so these updated classes are available on main

**Review by luluwuwu — APPROVED** [review]
approved

**Pull request merged**